airflow/testfiles/airflow_settings.yaml

```diff
@@ -0,0 +1,23 @@
+# This file allows you to configure Airflow Connections, Pools, and Variables in a single place for local development only.
+# NOTE: If putting a dict in conn_extra, please wrap in single quotes.
+
+# For more information, refer to our docs: https://www.astronomer.io/docs/cloud/stable/develop/customize-image#configure-airflowsettingsyaml
```

Comment:
Is this the right link? its point to the astro architecture

Reply:
yeah this link is good